distributed/deploy/spec.py

```diff
@@ -99,6 +99,44 @@ async def finished(self):
     def __repr__(self):
         return f"<{type(self).__name__}: status={self.status}>"
 
+    def _repr_html_(self):
+        self.address = getattr(self, "address", None)
+        self.external_address = None
+        self.lock = asyncio.Lock()
+        self.status = Status.created
+        self._event_finished = asyncio.Event()
```

Comment:
Thanks @freyam! It looks like these attributes are defined in the `__init__(...)` method and we probably don't want to overwrite them here as `_repr_html_` should just display existing state instead of mutating existing state

Reply:
Oops! I had commented out this code. Not sure how I let it pass through! 😄 Will fix it in the next commit.